app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -41,6 +47,17 @@ def backfill_transactions
       def configuration_params
         params.require(:super_good_solidus_taxjar_configuration).permit(:preferred_reporting_enabled)
       end
+
+      def api_sync
+        if ENV["TAXJAR_API_KEY"]
+          begin
+            yield
+          rescue Taxjar::Error => exception
+            flash[:error] = exception.message
```

Comment:
Should we provide some prefix to this message, like "Failed to complete request to TaxJar: ...`. The exception message here might not contain anything meaningful to an end user.